README.md

```diff
@@ -18,46 +18,49 @@
+## Guide
 
-***deviceShifu*** provides you with a high-level abstraction to interact with your device. By implementing the interface of the ***deviceShifu***, your IoT device can achieve everything it is designed for, and much more! For example, you can have your device actively push its telemetry to any endpoint of your choice.
+### Install
 
-## Start ***Shifu***
-
-### Installation
-
-***Shifu*** provides `shifu_install.yml`. If you have started a Kubernetes cluster, use the command `kubectl apply` to install ***Shifu*** in your cluster:
+If you have started a Kubernetes cluster, use the command `kubectl apply` to install ***Shifu*** in your cluster:
 
 ```sh
 cd shifu
 kubectl apply -f pkg/k8s/crd/install/shifu_install.yml
 ```
 
-### Demo
-
-If you are not familiar with Kubernetes, we provide [***Shifu*** Demo](https://shifu.run/disclaimer), which will intuitively show how ***Shifu*** creates and manages digital twins of any physical device in the real world.
-
-### Documentation
-
-See documentation on <https://shifu.run/docs/>.
+### Try it out
 
-## Look into ***Shifu***
+We present you [***Shifu*** Demo](https://shifu.run/disclaimer). 
 
-Check the Markdown files in [`docs/`](./docs/) to learn about the [design](./docs/design/) and [development guides](./docs/development/) of ***Shifu***.
+The Demo will show you how ***Shifu*** creates and manages digital twins of any physical devices.
 
-# ***Shifu***'s vision
-
-## Make developers and operators happy again
+### Documentation
 
-Developers and operators should focus entirely on using their creativity to make huge impacts, rather than fixing infrastructure and reinventing the wheel on a daily basis. As a developer and an operator, ***Shifu*** knows your pain!!! ***Shifu*** wants to take out all the underlying mess, and make us developers and operators happy again!
+For more information please visit our [documentation](https://shifu.run/docs/).
 
-## Software-Defined World
+## Design principle
 
-If every IoT device has a ***deviceShifu*** with it, we can leverage software to manage our surroundings, and make the whole world software defined. In a software-defined world, everything is intelligent, and the world around you will automatically change itself to serve your needs -- after all, all the technology we built is designed to serve human beings.
+Our job is to make developers and operators happy. Which is why all our designs are developer-oriented:
+#### 📡Easy to deploy
+Shifu can always be deployed with a single command.
+#### 🤖Plug'n'Play
+Shifu should automatically recognize and provide basic functionalities to a supported IoT device. Once the developer completes Shifu's template, all features of the device should be immediately available.
```

Comment:
`Shifu should automatically recognize and provide basic functionalities to a supported IoT device` -> `Shifu will automatically recognize and provide basic functionalities to a supported IoT device`